Stanford/JumpstartEngineering/Install/Courses/EnableCourses.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @file
+ * Enable Courses
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Courses;
+
+/**
+ *
+ */
+class EnableCourses extends \ITasks\AbstractInstallTask {
+
+  /**
+   * Enable Courses on JSE sites
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    module_load_include('module', 'redirect', 'redirect.module');
+    module_load_include('module', 'features', 'features.module');
+    $modules = array('stanford_feeds_helper', 'stanford_courses',
+      'stanford_course_views', 'stanford_courses_administration');
+
+    if (module_enable($modules, TRUE)) {
+      // Todo: revert stanford_course_views
+      //features_revert(array('stanford_course_views' => array('views_view')));
+      //features_revert_module('stanford_course_views');
+      drush_log('Enabled modules: ' . implode(', ', $modules), 'ok');
+    } else
+    {
+      drush_log('Error when enabling modules: ' . implode(', ', $modules), 'notice');
+      exit();
+    }
+
+    // Unpublish courses node
+    $query = new \EntityFieldQuery();
+
+    $entities = $query->entityCondition('entity_type', 'node')
+                      ->propertyCondition('type', 'stanford_page')
+                      ->propertyCondition('title', 'Courses')
+                      ->execute();
+
+    if (!empty($entities['node'])) {
```

Comment:
So, what is happening here is that _if_ the courses node exists it will keep the courses views from displaying and I want to get it it out of the way. I really _don't care_ if it doesn't exist. 
There's really no need to throw an exception if it doesn't exist.
Given that, how do you recommend I proceed?